docs/csharp/language-reference/keywords/explicit.md

```diff
@@ -10,11 +10,13 @@ ms.assetid: cfb8f42a-e411-4db2-af9b-796b05644846
 ---
 # explicit (C# Reference)
 
-The `explicit` keyword declares a user-defined type conversion operator that must be invoked with a cast. For example, this operator converts from a class called Fahrenheit to a class called Celsius:
+The `explicit` keyword declares a user-defined type conversion operator that must be invoked with a cast.
+
+The following example defines the operator that converts from a `Fahrenheit` class to a `Celsius` class (the operator must be defined either inside a `Fahrenheit` class or a `Celsius` class):
```

Comment:
It would be best to make "(the operator must be defined..." a separate sentence.